ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/CustomerRecommendationsResult.kt

```diff
@@ -0,0 +1,19 @@
+package com.braintreepayments.api.shopperinsights.v2
+
+import com.braintreepayments.api.core.ExperimentalBetaApi
+
+/**
+ * Customer recommendations for what payment options to show.
+ *
+ * @property sessionId The session ID for the customer session.
+ * @property isInPayPalNetwork Whether the customer is in the PayPal network.
+ * @property paymentRecommendations The payment recommendations for the shopper.
+ *
+ * Warning: This feature is in beta. It's public API may change or be removed in future releases.
+ */
+@ExperimentalBetaApi
+data class CustomerRecommendationsResult(
+    val sessionId: String?,
```

Comment:
Is the session ID ever not returned or do we always get one back?

Reply:
I believe we should always get a session ID back from the API, but in the GQL spec, it's defined as optional. So to match the spec, I defined `sessionId` as nullable.